src/plugins/controls/public/controls/data_controls/range_slider/components/range_slider.styles.ts

```diff
@@ -67,15 +61,15 @@ export const rangeSliderControlStyles = (euiThemeContext: UseEuiTheme) => {
         &:placeholder-shown,
         &::placeholder {
           font-weight: ${euiTheme.font.weight.regular};
-          color: ${euiTheme.colors.subduedText};
+          color: ${euiTheme.colors.textSubdued};
         }
       `,
       invalid: css`
         &:not(:invalid) {
-          color: ${euiTheme.colors.warningText};
+          color: ${euiTheme.colors.textWarning};
         }
         &:invalid {
-          color: ${euiTheme.colors.dangerText};
+          color: ${euiTheme.colors.textDanger};
```

Comment:
These styles appear to be broken 🤔 They're broken on `main` too, though, so I'm leaving it up to you whether you'd prefer to fix them in this PR or we could open a separate issue.

Reply:
I'll open a separate issue to fix the broken styles and just merge these changes for now

Reply:
Added it to my bug list in #https://github.com/elastic/kibana/issues/204593.